benchmarks/TFLite/CMakeLists.txt

```diff
@@ -91,6 +91,32 @@ iree_benchmark_suite(
     "dylib-sync"
 )
 
+# CPU, Dylib, 1-thread, big/little-core, full-inference. These benchmarks only
+# configure IREE translation and runtime flags for the target architecture and
+# runtime characteristics and do *not* include any non-default flags. No
+# non-default flags should be added here.
+iree_benchmark_suite(
+  MODULES
+    "${DEEPLABV3_FP32_MODULE}"
+    "${MOBILESSD_FP32_MODULE}"
+    "${POSENET_FP32_MODULE}"
+
+  BENCHMARK_MODES
+    "1-thread,big-core,full-inference,default-flags"
```

Comment:
I think `default-flags` is okay. But still wanted to hear your thoughts and have a bit bikeshedding here given that these names are kinda sticky once set. Comparing `default-flags`, have you thought about other tags? Like `default-features`/`mainstream` (vs. `beta-features`)? How do you think?

Reply:
I also considered just "default". Was considering "tuned" for the other configuration. I'm fine with any of those options (and agree that it makes sense to get this right the first time :slightly_smiling_face:)

Reply:
Hmm, I'm not able to make better names suggestions here. I kinda think `default-features` would be better because it doesn't anchor on flags too much, and we can contrast with `beta-features`. But `features` OTOH is quite generic and can be interpreted differently. So `default-flags` is fine I guess. Please feel free to land. :)

Reply:
Yeah I keep going back and forth on which is better. I think I'll just stick with flags